### PR TITLE
[MRG] Make parameter names in view_connectome() consistent with plot_connectome()

### PR DESCRIPTION
**Pull request by kchawla-pi**
For issue #1886 

Deprecated view_connectome params that differed from params of plot_connectome, docstrings & tests pending.

### CODE REVIEW

File: nilearn/plotting/html_connectome.py [hunk 1]
```diff
@@ -134,13 +148,51 @@ def view_connectome(adjacency_matrix, coords, threshold=None,
...
+                             'threshold': 'edge_threshold',
+                             'cmap': 'edge_cmap',
+                             'marker_size': 'node_size',
+                             }
+    used_deprecated_params = set(kwargs.keys()
+                                 ).intersection(all_deprecated_params.keys())
+    for deprecated_param_ in used_deprecated_params:
+        replacement_param = all_deprecated_params[deprecated_param_]
+        param_deprecation_msg = (
+            'The parameter "{}" will be removed in a future version of Nilearn.'
```
Comment (bthirion) [inline nilearn/plotting/html_connectome.py]: Shouldn't we state which version ?

Comment (kchawla-pi) [reply nilearn/plotting/html_connectome.py]: For Python2 deprecation we hadn't so I didn't here either.
From what I understand we display deprecation warnings for 2 releases and start clarifying the feature etc will be removed in the next release.

I assume we will handle this similarly.
I am happy to add a specific version now, if you tell me which one it should be. Nilearn 0.7.0? Or do we include point releases in next version?

Comment (bthirion) [reply nilearn/plotting/html_connectome.py]: I'd go for 0.7.0

Comment (jeromedockes) [reply nilearn/plotting/html_connectome.py]: the sooner we make the change the less code it will break

**Comment by kchawla-pi** [comment]
Yeah I'll fix this on Tuesday when I'm back.
@jeromedockes if you have feedback regarding how soon the deprecated parameters should be removed, let us know.

**Comment by kchawla-pi** [comment]
@bthirion
 @KamalakerDadi mirrored what  @jeromedockes has mentioned here.
The function wa introduced in Nilearn 0.5.0 so not a lot of people will be using it.
 believe even less will be using keyword arguments with it.
I don't think it will make much difference whether we remove the old parameter names in 0.6 or 0.7. What do you want?

**Comment by bthirion** [comment]
0.6 is fine  if everybody feels it's better.

**Comment by jeromedockes** [comment]
> I don't think it will make much difference whether we remove the old parameter names in 0.6 or 0.7. What do you want?

I think I would go for 0.6 but it's really up to you!

**Comment by kchawla-pi** [comment]
I am now implementing the change in default colormap. How should I implement that? I am going with always showing the warning for now, but I think there is space for some discussion about more nuance here.

**Comment by jeromedockes** [comment]
> I am now implementing the change in default colormap. How should I implement that? I am going with always showing the warning for now, but I think there is space for some discussion about more nuance here.

I'm not sure you need a warning for that; you can just change it and add a note in the whats_new and the docstring. if someone doesn't like the new colors they will easily figure out that the default colormap has changed.

**Comment by kchawla-pi** [comment]
Once the tests are green I will merge this tomorrow.

**Comment by jeromedockes** [comment]
do you plan to  address the similar issue with view_markers in a separate PR?

**Comment by jeromedockes** [comment]
I see you have kept the same default colormap, is that consistent with plot_connectome?

**Comment by kchawla-pi** [comment]
No, i noticed I hadn't changed it after posting this. I will change the colormap before merging.

**Comment by kchawla-pi** [comment]
> do you plan to address the similar issue with view_markers in a separate PR?

Yeah I am working on that one as we speak.

**Comment by jeromedockes** [comment]
I think tests outside of test_params_deprecation_view_connectome should use the new parameter names, e.g. test_view_connectome uses marker_size

**Comment by kchawla-pi** [comment]
Yeah, that should have changed by the IDE everywhere I'll do that.

**Comment by kchawla-pi** [comment]
This seems ready to be merge. Is it, @jeromedockes ?

**Comment by jeromedockes** [comment]
LGTM

**Pull request merged**